packages/control-property-editor/src/App.tsx

```diff
@@ -92,9 +93,14 @@ export default function App(appProps: AppProps): ReactElement {
         setWarningDialogVisibility(false);
     }
 
+    function closeAdpWarningDialog(): void {
+        setShouldShowDialogMessage(false);
+    }
+
     useEffect(() => {
         if (dialogMessage && isAdpProject) {
-            setShouldShowDialogMessageForAdpProjects(true);
+            setShouldShowDialogMessage(true);
+            setShouldHideIframe(dialogMessage.shouldHideIframe);
         }
     }, [dialogMessage]);
```

Comment:
`isAdpProject` should also be included in the dependency array for the `useEffect`
```suggestion
    }, [dialogMessage, isAdpProject]);
```

Reply:
Done